src/Components/Web.JS/package.json

```diff
@@ -23,12 +23,15 @@
     "jest": "^24.1.0",
     "ts-jest": "^24.0.0",
     "ts-loader": "^4.4.1",
-    "typescript": "^3.4.0",
+    "typescript": "^3.5.3",
     "webpack": "^4.12.0",
     "webpack-cli": "^3.0.8"
   },
   "resolutions": {
     "**/js-yaml": "^3.13.1",
-    "jest/**/handlebars": "^4.1.2"
+    "jest/**/handlebars": "^4.1.2",
+    "**/lodash": "^4.17.12",
+    "**/mixin-deep": "^1.3.2",
+    "**/set-value": "^2.0.1"
```

Comment:
I'm guessing you verified that there isn't a newer version of `jest` that natively has good dependencies, is that right?

Reply:
You are right. I updated all dependencies to the latest version and removed any unnecessary resolution override